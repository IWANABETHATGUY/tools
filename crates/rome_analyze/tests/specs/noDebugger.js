const test = { debugger: 1 }; test.debugger;

if (foo) debugger