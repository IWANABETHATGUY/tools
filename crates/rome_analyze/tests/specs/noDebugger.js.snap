---
source: crates/rome_analyze/tests/spec_tests.rs
assertion_line: 96
expression: noDebugger.js
---
# Input
```js
const test = { debugger: 1 }; test.debugger;

if (foo) debugger
```

# Diagnostics
```
warning[noDebugger]: This is an unexpected use of the debugger operator.
  ┌─ noDebugger.js:3:10
  │
3 │ if (foo) debugger
  │          -------- This is an unexpected use of the debugger operator.

Replace with emptyStatement
    | @@ -1,3 +1,3 @@
0 0 |   const test = { debugger: 1 }; test.debugger;
1 1 |   
2   | - if (foo) debugger
  2 | + if (foo) ;


```


